content/messaging/agent-chat-api/rtm-reference/index.mdx

```diff
@@ -557,248 +593,274 @@ Custom event is an event with customizable payload.
 
 System message event is native system event sent in specific situations.
 
-| Field                 | Req./Opt. |                                                                                          Notes |
-| --------------------- | :-------: | ---------------------------------------------------------------------------------------------: |
-| `created_at`          | required  |                           Date & time format with a resolution of microseconds, `UTC string`.  |
-| `type`                | required  | `system_message`                                                                               |
-| `system_message_type` | required  |                                                                                                |
-| `custom_id`           | optional  |                                                                                                |
-| `recipients`          | optional  | It can be specified when sending system messages via the [**Send Event**](#send-event) method. Possible values: `all`, `agents`.|
+### Request
+
+| Field                 | Required  | Data type | Notes                                                               | 
+| --------------------- | --------- |---------- | ---------------------------------------------------------------------------------------------: |
+| `custom_id`           | no        | `string`  | You can give the system message a custom ID.                                                                                               |
+| `type`                | yes       | `string`  | `system_message`                                                                               |
+| `text`                | no        | `string`  | ? |
```

Comment:
This text will override the default text displayed for each system msg type? Or is it some additional text?